tools/asset_builder/targets.txt

```diff
@@ -4,4 +4,6 @@ project/models/beeper.fbx
 project/models/coin.fbx
 project/models/table.fbx
 project/models/token.fbx
-project/models/worm.fbx
\ No newline at end of file
+project/models/worm.fbx
+project/models/Cube.fbx
```

Comment:
Is this a different "cube" than the one in nc/resources? I forgot to move the plane out of the project dir, but I'd like to keep the primitives in the nc/resources directory because the engine needs cube/sphere/plane regardless of the project.